flink-python/setup.py

```diff
@@ -252,7 +253,7 @@ def extracted_output_files(base_dir, file_path, output_directory):
                   "is complete, or do this in the flink-python directory of the flink source "
                   "directory.")
             sys.exit(-1)
-    if VERSION.find('dev0') != -1:
+    if re.search('dev.*$', VERSION) is not None:
```

Comment:
nit: Probably this can be argued, so feel free to ignore but in general I think `if <condition>:` is more readable. (And objectively more concise :)).

Reply:
I see example for both, I leave it like this until an additional +1 :)